x-pack/plugins/ml/public/application/timeseriesexplorer/timeseriesexplorer_embeddable_chart/timeseriesexplorer_embeddable_chart.js

```diff
@@ -242,6 +247,73 @@ export class TimeSeriesExplorerEmbeddableChart extends React.Component {
     this.props.appStateHandler(APP_STATE_ACTION.SET_ZOOM, zoomState);
   };
 
+  loadAnomaliesTableData = (earliestMs, latestMs) => {
+    const {
+      dateFormatTz,
+      selectedDetectorIndex,
+      selectedJob,
+      tableInterval,
+      tableSeverity,
+      functionDescription,
+    } = this.props;
+    const entityControls = this.getControlsForDetector();
+
+    return ml.results
+      .getAnomaliesTableData(
+        [selectedJob.job_id],
+        this.getCriteriaFields(selectedDetectorIndex, entityControls),
+        [],
+        tableInterval,
+        tableSeverity,
+        earliestMs,
+        latestMs,
+        dateFormatTz,
+        ANOMALIES_TABLE_DEFAULT_QUERY_SIZE,
+        undefined,
+        undefined,
+        functionDescription
+      )
+      .pipe(
+        map((resp) => {
+          const anomalies = resp.anomalies;
+          const detectorsByJob = mlJobService.detectorsByJob;
```

Comment:
Not sure how I feel about reintroducing a dependency on `mlJobService` if the intent is to deprecate it 🤔 
Perhaps it's method, `processBasicJobInfo`, which builds the `detectorsByJob` and `customUrlsByJob` can be moved to a new service that has no dependency on dependency cache and instead accepts `mlApiServices` and `notification` service?

I'm not sure how much of this work I've mentioned might be outside the scope of this PR - just thinking that we'll end up having to make these changes anyway so it might be worth getting a jump on it.

Happy to hear thoughts - @walterra, @jgowdyelastic, @peteharverson

Reply:
All the new lines here are brought over from `timeseriesexplorer.js` as is which this file is a duplicate of, it was just not part of the original embeddable duplicate because the chart itself didn't require the anomaly table data (which the anomaly marker actions popup does), so this should be cleaned up as part of the follow ups to consolidate `timeseriesexplorer_embeddable_chart.js` and `timeseriesexplorer.js`.